vsintegration/src/FSharp.Editor/Commands/XmlDocCommandService.fs

```diff
@@ -46,6 +46,7 @@ type internal XmlDocCommandFilter
 
     interface IOleCommandTarget with
         member _.Exec(pguidCmdGroup: byref<Guid>, nCmdID: uint32, nCmdexecopt: uint32, pvaIn: IntPtr, pvaOut: IntPtr) =
+            let mutable commitCharacter = true
```

Comment:
```suggestion
            let mutable isCommitCharacter = true
```

Reply:
No, it's not is, it's the flag whether should we commit character or not

Reply:
Right, okay. So maybe "shouldCommitCharacter" then :)